tests/e2e/specs/upe-split/shopper/shopper-upe-enabled-all-flows.spec.js

```diff
@@ -30,8 +34,6 @@ describe( 'Enabled Split UPE', () => {
 		await shopperWCP.changeAccountCurrencyTo( 'USD' );
 		await shopperWCP.logout();
 		await merchant.login();
-		//disable SEPA
-		await merchantWCP.disablePaymentMethod( [ sepaPaymentMethod ] );
```

Comment:
I think we still need this step to clean up the side effects of the test. Any specific reason to remove this?

Reply:
Nice catch, caught me slippin'. Amended in be2f198194db25d1666038ca3ba83127f3cbc256. 👍